dep/which/thunk.nix

```diff
@@ -2,7 +2,7 @@
 let fetch = { private ? false, fetchSubmodules ? false, owner, repo, rev, sha256, ... }:
   if !fetchSubmodules && !private then builtins.fetchTarball {
     url = "https://github.com/${owner}/${repo}/archive/${rev}.tar.gz"; inherit sha256;
-  } else (import <nixpkgs> {}).fetchFromGitHub {
+  } else (import "/nix/store/qjg458n31xk1l6lj26c3b871d4i4is98-source" {}).fetchFromGitHub {
```

Comment:
Is this what the pinned nixpkgs evaluates to? Seems so.

Reply:
Yeah I did
```fish
for x in dep/*/thunk.nix; nix-thunk unpack (dirname $x); and result/bin/nix-thunk pack (dirname $x); end
```
to update the thunks